### ISSUE TITLE
Fix CLI interaction with python

### ISSUE THREAD

**Issue by ACowan0105** [virtualcell/vcell#159]

**Comment by jcschaff** [comment]
think that this one is resolved - see @CodeByDrescher and @jcschaff

**Comment by CodeByDrescher** [comment]
This interaction was improved, however further changes need to made that are blocked by the current state of the vcell_cli_utils python files. This will be opened as its own issue.
